docs/source/dcos-docker-cli.rst

```diff
@@ -54,6 +54,45 @@ To wait until DC/OS has finished installing, use the :ref:`the dcos-docker wait
 
 To use this cluster, it is useful to find details using the :ref:`the dcos-docker inspect <dcos-docker-inspect>` command.
 
+Using a custom Docker network
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default DC/OS clusters are launched on the ``docker0`` network.
```

Comment:
❓ `s/network/bridge network`.

Reply:
Didn't address this one as I think it is not relevant here.